app/src/main/java/edu/uwstout/p2pchat/ChatFragment.java

```diff
@@ -73,18 +80,25 @@ public void onClick(final View view)
                 viewModel.insertMessage(address, timestamp, true, "text/message", text);
 
                 binding.textInput.setText("");
+
+                //An inmemory file.
+                InMemoryFile file = new InMemoryFile(text);
+
+                //Send an in-Memory-File with the new text.
```

Comment:
Either split inMemoryFile up into its individual words or use as is; don't hyphenate 
```suggestion
                //Send an inMemoryFile containing the current text message to the other device.
```